group_vars/location_gruni73/networks.yml

```diff
@@ -0,0 +1,117 @@
+---
+
+# IPv4 10.31.156.0/24
+# IPv6 2001:bf7:830:a600::/56
+
+ipv6_prefix: "2001:bf7:830:a600::/56"
+
+networks:
+  - vid: 42
+    role: mgmt
+    prefix: 10.31.156.0/27
+    gateway: 1
+    dns: 1
+    ipv6_subprefix: 1
+    assignments:
+      gruni73-core: 1
+      gruni73-switch: 2
+# 5ghz uplink
+      gruni73-sama: 5
+      gruni73-zwingli: 6
+# local nearfield aps 2ghz
+      gruni73-nf-sw-2ghz: 10
+      gruni73-nf-o-2ghz: 11
+# local nearfield aps 5ghz
+      gruni73-nf-w-5ghz: 13
+      gruni73-nf-so-5ghz: 14
+
+  - vid: 40
+    role: dhcp
+    prefix: 10.31.156.128/25
+    ipv6_subprefix: 0
+    inbound_filtering: true
+    enforce_client_isolation: true
```

Comment:
If this location is meant to be public  only, please consider to deactivate these options. We had a consensus on the mailing-list, that these options should only be used for house projects.

Reply:
Cannot confirm we had a consensus on that, its up the the location maintainer as of now.

Reply:
> Filtering violates Rule 1 of the Pico Peering Agreement, which is the foundation of Freifunk. We had a quite broad discussion on that topic on the mailing list.

Thats an open question. Even people which contributed to the PPA stated that it solely applies to mesh traffic (Peering).

>  to tolerate this violation of the PPA in house projects
Stop calling this a violation!

> For the rest of the network, most users disagreed on any filtering.
"Most Users" ? We didnt ask the users. Responding People on Mailing List != Users

> the standard behavior is to not filter anything, as you mentioned here too:
Yes - If its not explicitly enabled in the config its disabled. If the maintainer wants to enable filtering it can be enabled.

Reply:
This snippet was just copy pasted, I'm happy to use whichever config people agree on. But if I were to take a stance, I think since this is mostly a public installation with no 'home' users I'd say the safer option is the better one.

Reply:
We had an out of band conversation about this on matrix, and there is no consensus. Since I'd like to deploy this site before consensus is reached, I'd like to keep the config as it is currently.

I am however very happy to revisit / abide by any community wide decisions about this that may come about later.

Reply:
I'd be fine with this. :)